src/tunacode/core/logging/sentry.py

```
@@ -0,0 +1,57 @@
+SENTRY_ENVIRONMENT_ENV = "SENTRY_ENVIRONMENT"
+DEFAULT_ENVIRONMENT = "production"
+
+_sentry_initialized: bool = False
+
+
+def init_sentry() -> bool:
+    """Initialize Sentry if DSN is configured and SDK is available.
+
+    Returns True if Sentry was successfully initialized, False otherwise.
+    """
+    global _sentry_initialized  # noqa: PLW0603
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLW0603`)

Remove unused `noqa` directive

(RUF100)